### DISCUSSION TITLE
pybricks.experimental import getchar

### DISCUSSION THREAD

**Discussion by alpha71** [pybricks/support#453] [General]
Hello,
i tried use getchar from the experimental module but i got error:

```python
from pybricks.hubs import TechnicHub
from pybricks.tools import wait
from pybricks.parameters import Color
from pybricks.experimental import getchar

hub = TechnicHub()

while True:
    c = getchar()
    if c == ord('q'):
        hub.light.on(Color.GREEN)
        wait(30)
    elif c == ord('a'):
        hub.light.on(Color.YELLOW)
        wait(30)
    else:
        hub.light.on(Color.BLUE)
    wait(30)`
```
![immagine](https://user-images.githubusercontent.com/37776195/130939398-c10bb1da-df95-40da-9f86-8611c03c0f36.png)

Let me know if i'm doing something wrong! Thank you
Ale

**Comment by laurensvalk** [comment]
This functionality was moved.

> You can now read from `stdin` to achieve the same result. We have an example project [here](https://pybricks.com/projects/sets/technic/42099-off-roader/keyboard-remote/). Now that this feature is available, we have deprecated `getchar`.

Source: https://github.com/pybricks/support/issues/376#issuecomment-871127130.

**Reply by alpha71** [reply]
Hello Laurens, 
I tried and it works. Very interesting feature, thank very much for your support.
Bye
Ale
